server.js

```diff
@@ -148,7 +148,46 @@ app.post('/PostEditAnime', async function (req, res) {
 			return res.status(400).json({err: 'Title is missing'});
 		}
 	} catch (error) {
-		console.log('/PostAddAnime Error');
+		console.log('/EditAnime Error');
+		return res.status(501);
+	}
+});
+
```

Comment:
There is a lack of protection against entering the wrong number of episodes and an error code message (also in /PostAddAnime).